src/index.ts

```diff
@@ -6,11 +6,8 @@ export type Amount = {
 };
 
 export type CardConfiguration = {
-  /** @todo NOT IMPLEMENTED */
   showsHolderNameField?: boolean;
-  /** @todo NOT IMPLEMENTED */
   showsStorePaymentMethodField?: boolean;
-  /** @todo NOT IMPLEMENTED */
   showsSecurityCodeField?: boolean;
```

Comment:
Since you check if `card` is set and then force-unwrap the properties you should remove `?` from these three

Reply:
Good point. Updated now.